opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -422,9 +422,15 @@ namespace Opm
 
         // initialized the well rates with the potentials i.e. the well rates based on bhp
         const int np = this->number_of_phases_;
-        const double sign = well_copy.well_ecl_.isInjector() ? 1.0 : -1.0;
+        bool trivial = true;
         for (int phase = 0; phase < np; ++phase){
-            ws.surface_rates[phase] = sign * ws.well_potentials[phase];
+            trivial = trivial && (ws.well_potentials[phase] == 0.0) ;
+        }
+        if (!trivial) {
+            const double sign = well_copy.well_ecl_.isInjector() ? 1.0 : -1.0;
+            for (int phase = 0; phase < np; ++phase) {
+                ws.surface_rates[phase] = sign * ws.well_potentials[phase];
+            }
         }
```

Comment:
So this will leave `ws.surface_rates` unchanged if the `trivial` condition is true.  Is there any guarantee that those `surface_rates` values will be reasonable/usable in that case?

Reply:
> Is there any guarantee that those surface_rates values will be reasonable/usable in that case?

The purpose of the rest of the commit is to assure usable initial condition for the case where the well start's up with zero rate and potentials.

Reply:
>> Is there any guarantee that those surface_rates values will be reasonable/usable in that case?
>
> The purpose of the rest of the commit is to assure usable initial condition for the case where the well start's up with zero rate and potentials.

Ah, right.  Sorry for not skimming though the complete PR before commenting.